src/Core/Identity/WebAuthnTokenProvider.cs

```diff
@@ -65,14 +65,13 @@ public async Task<string> GenerateAsync(string purpose, UserManager<User> manage
 
             var exts = new AuthenticationExtensionsClientInputs()
             {
-                UserVerificationIndex = true,
                 UserVerificationMethod = true,
                 AppID = CoreHelpers.U2fAppIdUrl(_globalSettings),
             };
 
             var options = _fido2.GetAssertionOptions(existingCredentials, UserVerificationRequirement.Discouraged, exts);
 
-            provider.MetaData["login"] = options;
+            provider.MetaData["login"] = JsonSerializer.Serialize(options);
```

Comment:
I believe this can be removed once everything is updated to use system.text.json.

Reply:
Now that we've migrated to system.test.json, I'll need to check this.

Reply:
Still needed, we still serialize providers using newtonsoft